docs/source/overview/deployment.rst

```diff
@@ -0,0 +1,303 @@
+
+.. _continuous_integration_of_policies:
+
+Continuous Integration of Policies
+----------------------------------
+
+Next, enable a CI webhook back to your CI system of choice when pull requests
+targeting your master branch are opened or updated. This allows us to continuously
+test and validate the policies that are being modified.
+
+In this example, we will be using Microsoft Azure Devops Pipelines.
+
+First, navigate to https://azure.microsoft.com/en-us/services/devops/pipelines/ and
+click the "Start pipelines free with Github" button and follow the flow to connect
+your Github account with Devops Pipelines.
+
+Next click on the Piplines section in the left hand side of the sidebar and connect
+with Github. Once the pipeline is setup, we can add the following azure devops
+configuration to our repo:
+
+.. code-block:: yaml
+
+    trigger:
+    - master
+    
+    jobs: 
+      - job: 'Validate'
+        pool:
+          vmImage: 'Ubuntu-16.04'
+        steps:
+          - checkout: self
+          - task: UsePythonVersion@0
+            displayName: "Set Python Version"
+            inputs:
+              versionSpec: '3.7'
+              architecture: 'x64'
+          - script: pip install --upgrade pip
+            displayName: Upgrade pip
+          - script: pip install c7n c7n_azure c7n_gcp
+            displayName: Install custodian
+          - script: custodian validate policy.yml
+            displayName: Validate policy file
+
+This configuration will install Cloud Custodian and validate the policy.yml file
+that we created in the previous step.
+
+Finally, we can run the new policies against your cloud environment in dryrun mode.
+This mode will only query the resources and apply the filters on the resources. Doing
+this allows you to assess the potential blast radius of a given policy change.
+
+Setting up the automated dryrun of policies is left as an exercise to the user-- this
+requires hosting your cloud authentication tokens inside of a CI system or hosting your
+own CI system and using Managed Service Identities (Azure) or Instance Profiles (AWS).
+
+It's important to verify that the results of the dryrun match your expectations. Custodian
+is a very powerful tool that will do exactly what you tell it to do! In this case, you should
+always "measure twice, cut once".
+
+.. _iam_setup:
+
+IAM Setup
+---------
+
+To run Cloud Custodian against your account, you will need an IAM role with appropriate
+permissions. Depending on the scope of the policy, these permissions may differ from policy
+to policy. For a baseline, the managed read only policies in each of the respective cloud
+providers will be enough to dryrun your policies. Actions will require additional IAM
+permissions which should be added at your discretion.
+
+For serverless policies, Custodian will need the corresponding permissions to provision
+serverless functions.
+
+In AWS, you will need ReadOnly access as well as the following permissions:
```

Comment:
I'd like to see some more examples for other clouds here, i'm just not familiar with them

Reply:
@aluong can you follow up on this in another pr?